zest/releaser/prerelease.py

```diff
@@ -127,6 +132,24 @@ def _grab_history(self):
         # TODO: add line number where an extra changelog entry can be
         # inserted.
 
+        # Look for 'Nothing changed yet' under the latest header.  Not
+        # nice if this text ends up in the changelog.  Did nothing happen?
+        start = headings[0]['line']
```

Comment:
Yeah, some of these pull requests touch similar code, and of course the CHANGES.rst as well for conflicts. So I'll keep my hands of it for now if you are busy with it.
It was a productive Plone Open Garden in Sorrento last week for zest.releaser. :-)